pyproject.toml

```diff
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools>=61.2.0", "wheel", "setuptools_scm[toml]>=3.4.3"]
```

Comment:
Does this mean the end user needs to have these tools installed or just the person who builds the dist file and wheel?